packages/terra-form-checkbox/src/Checkbox.jsx

```diff
@@ -83,131 +83,97 @@ const defaultProps = {
   value: undefined,
 };
 
-class Checkbox extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { focus: false };
-    this.onFocus = this.onFocus.bind(this);
-    this.onBlur = this.onBlur.bind(this);
-  }
-
-  onFocus() {
-    this.setState({ focus: true });
-    if (this.props.onFocus) {
-      this.props.onFocus();
-    }
+const Checkbox = ({
+    checked,
+    defaultChecked,
+    inputAttrs,
+    id,
+    isDisabled,
+    isInline,
+    isLabelHidden,
+    labelText,
+    labelTextAttrs,
+    name,
+    onBlur,
+    onChange,
+    onFocus,
+    value,
+    ...customProps
+  }) => {
+  const controlInputAttrs = Object.assign({}, inputAttrs);
+
+  if (checked !== undefined) {
+    controlInputAttrs.checked = checked;
+  } else {
+    controlInputAttrs.defaultChecked = defaultChecked;
   }
 
-  onBlur() {
-    this.setState({ focus: false });
-    if (this.props.onBlur) {
-      this.props.onBlur();
-    }
+  const checkboxClasses = cx([
+    'checkbox',
+    { 'checkbox-inline': isInline },
```

Comment:
Can we be consistent with class names? since you have is-disabled, I think is-inline would be appropriate.

Reply:
+1